packages/shared/components/ColorPicker.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="typescript">
```

Comment:
Super confusing file, with the various states the styles can be in. Would be good to clean this up in the future and maybe separate the custom picker as its own component.